manager/backend/api.py

```diff
@@ -198,52 +197,128 @@ def destroy(self, request, author_id=None, id=None, *args, **kwargs):
 			# Return the serializer output data as the response
 			return Response(deleted_post)
 		return super().destroy(request, *args, **kwargs)
+		
+	def create(self, request, author_id=None, id=None, *args, **kwargs):
+		if author_id and id:
+			post = Post(
+			author = Author.objects.filter(id=author_id).get(),
+			id = id,
+			title = request.data["title"],
+			source = request.data["source"],
+			origin = request.data["origin"],
+			host = self.request.META['HTTP_HOST'],
+			description = request.data["description"],
+			content_type = request.data["contentType"],
+			content = request.data["content"],
+			categories = request.data["categories"],
+			count = 0,
+			visibility = request.data["visibility"],
+			unlisted = request.data["unlisted"]
+		)
+		elif author_id:
+			post = Post(
```

Comment:
what's the difference between this and the above if code block? Can we shorten it to just `if author_id`